hw/bsp/rp2040/family.cmake

```diff
@@ -23,133 +23,218 @@ if (NOT TARGET _rp2040_family_inclusion_marker)
 		set(PICO_TINYUSB_PATH ${TOP})
 	endif()
 
-	if (NOT TINYUSB_OPT_OS)
-		set(TINYUSB_OPT_OS OPT_OS_PICO)
+	unset(SUPPORTED_OS_LIST)
+	if (DEFINED TINYUSB_OPT_OS)
+		# for backward compatibility, let the user force the OS option
+		list(APPEND SUPPORTED_OS_LIST ${TINYUSB_OPT_OS})
+	else()
+		# otherwise, build both the pico-sdk and FreeRTOS versions of the library
+		list(APPEND SUPPORTED_OS_LIST OPT_OS_PICO OPT_OS_FREERTOS)
 	endif()
+	set(PROJ_SUFFIX "")
+	foreach(SUPPORTED_OS IN LISTS SUPPORTED_OS_LIST)
```

Comment:
why did you copy this here when you have FreeRTOS_Kernel_import.cmake too?

Reply:
Because FreeRTOS_Kernel_import.cmake has a fatal error message lines 47-49.
```
if (NOT FREERTOS_KERNEL_PATH)
    message(FATAL_ERROR "FreeRTOS location was not specified. Please set FREERTOS_KERNEL_PATH.")
endif()
```
As I understand it, we do not want pico-sdk to fail to build if FreeRTOS-Kernel is not installed.